### PR TITLE
attempt to fix the icons dir issue

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] fix the nagging icons folder issue
- [x] tested by running pnpm run dev few times on local machine

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "version": "3.0.0",
   "private": true,
   "scripts": {
-    "dev": "nuxi prepare && cross-env PORT=9090 nuxi dev",
+    "dev": "nuxi prepare && pnpm run delete-icons && cross-env PORT=9090 nuxi dev",
```
Comment (daiagi) [inline package.json:22]:
```suggestion
"dev": " pnpm run delete-icons && nuxi prepare && cross-env PORT=9090 nuxi dev",
```

**Review by roiLeo — COMMENTED** [review]
I don't think we need to remove two time `./.nuxt/pwa`

- `rm -rf ./.nuxt/pwa/*` will result the same as `rm -rf ./.nuxt/pwa`
- do we really need to `pnpm run delete-icons` on `pnpm dev` I never had this dir issue while using this command

**Comment by daiagi** [comment]
>     * `rm -rf ./.nuxt/pwa/*` will result the same as `rm -rf ./.nuxt/pwa`

I agree, but still right now it doesn't work
so at best, it may fix it, at worst it will do no harm

>     * do we really need to `pnpm run delete-icons` on `pnpm dev` I never had this dir issue while using this command

oh really?! i get it all the time... maybe OS related? I use Ubuntu, and you?

**Comment by roiLeo** [comment]
> oh really?! i get it all the time... maybe OS related? I use Ubuntu, and you?

OSX, I encoutered it one time when using `nuxi generate`

**Comment by vikiival** [comment]
What is the status of this issue? 

Can maybe @preschian check or so ?

**Comment by daiagi** [comment]
@preschian  guess close this pr then?

**Comment by vikiival** [comment]
Closing as suggested by prescian, approved by roiLeo.
Thanks for the quickfix @daiagi ^-^

**Pull request closed without merging**